### PR TITLE
Extend/fix mixer documentation

### PR DESCRIPTION
**Pull request by fiona-byte**

### CODE REVIEW

File: source/chapters/user_interface.rst [hunk 1]
```diff
@@ -871,8 +871,20 @@ Equalizers and Gain Knobs
...
 
 **Gain Knob**
+  Gain or Loss in sounds is also referred to as volume levels.
   The gain knob allows you to adjust the gain applied to the deck.
   Use this to compensate for the differences in recording levels between tracks.
+  There are 4 Gain Knobs:
+      1. The Super-Knob
+      2. The Low Filter Knob
+      3. The Mid Filter Knob
+      4. The High Filter Knob
```
Comment (ronso0) [inline source/chapters/user_interface.rst:881]:
```suggestion
  There are 4 knobs:
      1. Gain
      2. High filter
      3. Mid filter
      4. Low filter
```
This was wrong.
Also when referencing screenshots please describe elements in the same order the appear in the image.

File: source/chapters/user_interface.rst [hunk 2]
```diff
@@ -871,8 +871,20 @@ Equalizers and Gain Knobs
...
+  Each knob has a left and right side. The left side directs sounds to the left speakers or headset. The right side directs sounds to the right speakers or headset.
+   The mid-position places the filter in a balanced position. This allows the filter to be shared equally by both the left and right sides of the headset.
```
Comment (ronso0) [inline source/chapters/user_interface.rst:883]: wrong. EQ knobs affect the volume of certain frequency ranges. Balance is something totally different and balance controls are located only in the Master section of the mixer.
please use Mixxx yourself to test what the controls do and to pick additonal info from the tooltips (see above comment)

File: source/chapters/user_interface.rst [hunk 3]
```diff
@@ -871,8 +871,20 @@ Equalizers and Gain Knobs
...
+   The low filter knob regulates the bass.
+   The mid filter knob regulates the main voice.
+   The high filter knob regulates the treble from the sound.
+  These effects are best heard with the headset as a secondary output.
```
Comment (ronso0) [inline source/chapters/user_interface.rst:887]:
```suggestion
```

File: source/chapters/user_interface.rst [hunk 4]
```diff
@@ -926,8 +938,10 @@ Crossfader
    The crossfader section of the mixer
 
 The :term:`crossfader` determines the actual volume of each deck when moving
-the slider from left to right. If both decks are playing and the crossfader is
-in its default center position, then you will hear both decks. Right-clicking on
+the slider from left to right. Simply put, the crossfader aids the user to regulate the volume and the intensity of the sound output from both deck.
+Crossfading is a sound output effect that allows the user play more than one song at a time on different deck.
+It is a very useful function when making music mix. The user can slowly fadeout a song with the crossfader and allow the new song come up.
+If both decks are playing and the crossfader is in its default center position, then you will hear both decks. Right-clicking on
```
Comment (ronso0) [inline source/chapters/user_interface.rst:944]: this change is not necessary IMO.
instead add a section about crossfader assignment switches. those allow you to assign a deck to the left, right or center position of the crossfader.
you find them here:
2 decks
image
4 decks 
image

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by fiona-byte** [comment]
Alright. I'd make these changes

**Comment by Holzhaus** [comment]
@fiona-byte you don't need to open a new Pull Request, just `git push` your changes and the PR will be updated.

**Comment by fiona-byte** [comment]
Alright @Holzhaus . I'd make the necessary changes

**Comment by ronso0** [comment]
@fiona-byte Any updates on this? Any issues with `git` we should help with?

**Comment by ronso0** [comment]
If you like you can also pick up https://github.com/mixxxdj/manual/issues/238 because it's closely related and rather simple to fix.

**Comment by ronso0** [comment]
@fiona-byte 
I just assigned #238 to @ulomau

**Comment by Holzhaus** [comment]
Is this ready? The review comments are all outdated...

**Comment by ronso0** [comment]
Some comments still apply, see https://github.com/mixxxdj/manual/pull/215/files#diff-b69a44a7d04875bdb78b86f9b4c3d69a23de77ad080cd7897f1bce1b19b0588eR881-R883

@fiona-byte Do you plan to finish this?

**Comment by hannydevelop** [comment]
> @fiona-byte Do you plan to finish this?

Hello @ronso0, @Holzhaus and @fiona-byte should I finish this?

**Comment by ronso0** [comment]
Sure, would be appreciated. Some of my first review comments still apply.
I suggest to play with the features you intend to describe and also check the tooltips in the GUI for explanation.

**Comment by hannydevelop** [comment]
ok

**Comment by fiona-byte** [comment]
Hi @ronso0 @hannydevelop, you can go ahead and complete it.

**Comment by hannydevelop** [comment]
> Hi @ronso0 @hannydevelop, you can go ahead and complete it.

Ok @fiona-byte, I'll work on this then.

**Comment by ronso0** [comment]
@hannydevelop I suggest to start a new branch and consider the review comments posted here.

**Pull request closed without merging**